pinning-services/index.js

```diff
@@ -342,12 +342,17 @@ function updateDnsRecordDigitalOcean(domain, recordType, recordName, recordData,
       // Delete existing records with matching record type and name
       if (isDeleted) {
         txt.forEach((item, index) => {
-          const urlDelete = url + item['id'];
-          console.log(`DELETE ${urlDelete}`);
-          fetch(urlDelete, { method: 'DELETE', headers: headers })
-            .catch(function(error) {
-              console.log(error);
-            });
+          let isSkipDelete = 0;
+          if (item['type'] == 'TXT') isSkipDelete = 0;
+          if (item['type'] == 'TXT' && (item['data'].indexOf('datkey') == -1 && item['data'].indexOf('dnslink') == -1)) isSkipDelete = 1;
```

Comment:
Replace all with `===`

Reply:
We don't need the inner brackets. They are just 3 sets of conditions all needing to be true.

Reply:
You should put the `=` sign just to be safe, since `datkey` and `dnslink` itself can be valid value of the key.

I think all this above should just be:
```
isSkipDelete = item['type'] === 'TXT' && item['data'].indexOf('datkey=') === -1 && item['data'].indexOf('dnslink=') === -1)
```

Except we are already filtered for TXT records here. So I think you really only need one line in this patch?

```
if (item['data'].indexOf('datkey=') === -1 && item['data'].indexOf('dnslink=') === -1) break;
```